.github/workflows/validate-change-notes.yml

```
@@ -0,0 +1,31 @@
+name: Validate change notes
+
+on:
+  push:
+    paths:
+      - "*/ql/*/change-notes/**/*"
+      - ".github/workflows/validate-change-notes.yml"
+    branches:
+      - main
+      - "rc/*"
+  pull_request:
+    paths:
+      - "*/ql/*/change-notes/**/*"
+      - ".github/workflows/validate-change-notes.yml"
+
+jobs:
+  check-change-note:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup CodeQL
+        uses: ./.github/actions/fetch-codeql
+
+      - name: Fail if there are any errors with existing change notes
+        if: |
+          github.event.pull_request.draft == false
```

I think there is no need for skipping this check in draft mode. Personally, I like to be notified of simple errors as early as possible.

```suggestion
        if: |
          github.event.pull_request.draft == false
```

---

That's fine. I was just following the example of the check-change-note job. I have no strong feelings either way.

```suggestion
```